packages/kbn-cell-actions/src/components/cell_actions.tsx

```diff
@@ -53,14 +56,17 @@ export const CellActions: React.FC<CellActionsProps> = ({
   }
 
   return (
-    <div ref={nodeRef} data-test-subj={dataTestSubj}>
+    <EuiFlexGroup alignItems="center" ref={nodeRef} gutterSize="none" justifyContent="flexStart">
```

Comment:
I see `InlineActions` also wraps everything in `EuiFlexGroup`, do we need 2 of them?

Reply:
Yes, the `EuiFlexGroup` inside cell actions aligns the actions div with cell actions children props.  If you delete it, the component will look like this: 
<img width="114" alt="Screenshot 2023-02-08 at 15 42 25" src="https://user-images.githubusercontent.com/1490444/217573411-ee81340f-abfc-47e4-95d7-4086ef9ed046.png">

The `EuiFlexGroup` inside `InlineActions` aligns the actions inside the div. If you delete it, the component will look like this: 
<img width="137" alt="Screenshot 2023-02-08 at 16 26 14" src="https://user-images.githubusercontent.com/1490444/217574000-56f70f45-c0a7-48f8-85c2-e099ea4340d5.png">